src/lib/formats/tzx_cas.cpp

```diff
@@ -791,11 +900,20 @@ static int cdt_cas_fill_wave( int16_t *buffer, int length, uint8_t *bytes )
 {
 	int16_t *p = buffer;
 	int size = 0;
-	t_scale = (40 / 35);  /* scale to 4MHz */
 	size = tzx_cas_do_work(&p);
 	return size;
 }
 
+static int tsx_cas_fill_wave( int16_t *buffer, int length, uint8_t *bytes )
+{
+	int16_t *p = buffer;
+	int size = 0;
```

Comment:
What’s the purpose of this local?

Reply:
I have duplicated the "_cas_fill_wave" function for TSX, I don't know if you mean that

Reply:
I have unified the 3 extensions in the last change uploaded